articles/application-insights/app-insights-sampling.md

```diff
@@ -301,7 +302,7 @@ Fixed-rate and adaptive sampling are a feature of the SDK in ASP.NET versions fr
 
 The sampling algorithm decides which telemetry items to drop, and which ones to keep (whether it's in the SDK or in the Application Insights service). The sampling decision is based on several rules that aim to preserve all interrelated data points intact, maintaining a diagnostic experience in Application Insights that is actionable and reliable even with a reduced data set. For example, if for a failed request your app sends additional telemetry items (such as exception and traces logged from this request), sampling will not split this request and other telemetry. It either keeps or drops them all together. As a result, when you look at the request details in Application Insights, you can always see the request along with its associated telemetry items. 
 
-For applications that define "user" (that is, most typical web applications), the sampling decision is based on the hash of the user id, which means that all telemetry for any particular user is either preserved or dropped. For the types of applications that don't define users (such as web services) the sampling decision is based on the operation id of the request. Finally, for the telemetry items that neither have user nor operation id set (for example telemetry items reported from asynchronous threads with no http context) sampling simply captures a percentage of telemetry items of each type. 
+For applications that define "user" (that is, most typical web applications), the sampling decision is based on the hash of the user id, which means that all telemetry for any particular user is either preserved or dropped. (User id is removed from sampling calculations from 2.5.0-beta2 of .NET SDK, and 2.2.0-beta3 of .NET Core SDK) For the types of applications that don't define users (such as web services) the sampling decision is based on the operation id of the request. Finally, for the telemetry items that neither have user nor operation id set (for example telemetry items reported from asynchronous threads with no http context) sampling simply captures a percentage of telemetry items of each type. 
```

Comment:
We first say that sampling is based on user and then say user Id is removed. Can we restructure in a way that it first says that sampling is based on operation Id and that in versions prior 2.5-b2 it was based on user Id when available?
This way it will be a bit more clearer message for the current stable versions.